docs/words.tex

```diff
@@ -367,6 +382,19 @@ \section{System State}
 : x ;
 forget
 \end{verbatim}
+
+\item[traverse-wordlist ( xt -- )] Executes \texttt{xt} for successive headers starting from \texttt{latest}. Each time \texttt{xt} is called, it must consume an argument from the stack and leave behind a truth value. If true (not 0), it will continue to the next header address and call \texttt{xt} again. If false, or if it reaches the end of the dictionary, traverse-wordlist will return. Example:

```

Comment:
Idea!
Since `TRAVERSE-WORDLIST` is anyway not standard, how about calling it something shorter instead, that fits better on screen?
How do you like `DOWORDS`?